Automattic/wp-calypso
# Launchpad - Update launchpad domain upsell completion

Related to #74416

## Test / Review Estimation

Test: Medium
Review: Short

## Proposed Changes

* Adds logic to update site option `launchpad_checklist_tasks_statuses` with new value `domain_upsell_deferred: true` when the user accesses the domain upsell flow and clicks the "Choose a domain later" link on the domains page. `domain_upsell_deferred` will be used to cross off the `Choose my domain later` task on the launchpad screen.

![CleanShot 2023-03-30 at 16 37 41@2x](https://user-images.githubusercontent.com/10482592/228958583-93e10b95-6869-44c8-9000-d478c82674ce.png)


							

## Testing Instructions



* Checkout this branch
* Create a new free flow launchpad-enabled site
* Navigate to the launchpad screen and click the "Choose a domain" task.
* Click the "Choose my domain later" link on the domain selection screen. It should be on the right side of the screen.
* Verify the "Choose a domain" task is marked as completed on the launchpad screen

## Pre-merge Checklist
